IssueTracker/IssueTracker.FileSystem/Repository/FileProvider.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System.Runtime.CompilerServices;
+using Microsoft.Extensions.Configuration;
+
+[assembly: InternalsVisibleTo("IssueTracker.UnitTest")]
+namespace IssueTracker.FileSystem;
+public class FileProvider : IFileProvider
+{
+    private readonly IMetaDataProvider _metadata;
+    private readonly IBlobProvider _bolb;
+
+    public FileProvider(IConfiguration config)
+    {
+        IConfigurationFactory cf = new ConfigurationFactory(config);
+        var metaDataConfig = cf.Create<IMetaDataConfiguration>();
+        _metadata = new MetaDataProvider(metaDataConfig);
+        var blobConfig = cf.Create<IBlobConfigurationFactory>();
+        _bolb = new BlobData(blobConfig);
+    }
+
+    public async Task<bool> DeleteAsync(Models.File file)
+    {
+        var result = await _metadata.DeleteAsync(file.Id, file.Extension);
+        if (result == false) return false;
+        result = await _bolb.DeleteAsync(file.Id + file.Extension);
+        return result;
+    }
+
+    public async Task<IEnumerable<Models.File>> GetAsync(IEnumerable<Models.File> files)
+    {
+        var entities = _metadata.Get(files);
+        if (!entities.Any()) throw new ArgumentException("Detail file is null!");
```

Comment:
This should not be an ArgumentException.

Reply:
will change